spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Rule.java

```diff
@@ -41,11 +41,13 @@
     }
 
     JSONObject toJSONObject() {
-        JSONObject messageStrings = new JSONObject().put("default", new JSONObject().put("text", defaultText));
+        String textEndsWithPeriod = !defaultText.endsWith(".") ? defaultText + "." : defaultText;
```

Comment:
We have a pattern that each text should end with a period. So this check is doing that.

Reply:
"if not...else" is a double-negative; can this be reversed to be more easily readable?

Reply:
just updated! thanks for the review!